lib/http_request_helper.rb

```diff
@@ -0,0 +1,32 @@
+module HTTPRequestHelper
+  HTTP_METHODS = {
+    get: Net::HTTP::Get,
+    post: Net::HTTP::Post,
+  }.freeze
+
+  def default_request_response(url:, body:, headers:, type: :post)
+    uri = URI(url)
+    http = Net::HTTP.new(uri.host, uri.port)
+    http.use_ssl = (url.scheme == 'https')
+    request = HTTP_METHODS[type].new(uri.request_uri)
+    headers&.each { |key, val| request[key] = val }
+    request.body = body.to_json if body
+    response = http.request(request)
+
+    {
+      body: JSON.parse(response.read_body),
+      status: response.code.to_i,
+    }
+  rescue
+    {body: 'Error occured',
+     status: 503,}
```

Comment:
Rubocop reports styling issues, spaces missing. Apart from that, looking good!

Reply:
Hey, which version of rubocop.yml do you use? 
![image](https://user-images.githubusercontent.com/7688888/68567384-ae98eb80-047a-11ea-99bb-ea592de57a19.png)

Reply:
Oops, my bad. I've modified `Documentation` to `Style/Documentation`